src/runtime/audience-action-flow.js

```diff
@@ -253,6 +259,20 @@ export class AudienceActionFlow {
     }
   }
 
+  /**
+   * @param {SurveyDataTransferRequest} request
+   * @private
+   */
+  // eslint-disable-next-line no-unused-vars
+  handleSurveyDataTransferRequest_(request) {
+    // @TODO(justinchou): execute callback with
+    // setOnInterventionComplete and Google Analytics
```

Comment:
Can you run linter to arrange the comment lines?

Reply:
not sure if the formatter is smart enough to fix all formatting issues (as formatting the doc isn't changing anything for me), but I think you're asking to condense them a bit more? see changes

Reply:
Yeah, looks good. Thanks!